### PR TITLE
chore: lint typescript files

### PR DESCRIPTION
**Pull request by AbdelrahmanHafez**

### CODE REVIEW

File: types/index.d.ts [hunk 1]
```diff
@@ -939,20 +941,20 @@ declare module 'mongoose' {
...
-      T extends Buffer ? SchemaDefinition<typeof Buffer> :
-      T extends Types.ObjectId ? ObjectIdSchemaDefinition :
-      T extends Types.ObjectId[] ? AnyArray<ObjectIdSchemaDefinition> | AnyArray<SchemaTypeOptions<ObjectId>> :
-      T extends object[] ? (AnyArray<Schema<any, any, any>> | AnyArray<SchemaDefinition<Unpacked<T>>> | AnyArray<SchemaTypeOptions<Unpacked<T>>>) :
-      T extends string[] ? AnyArray<StringSchemaDefinition> | AnyArray<SchemaTypeOptions<string>> :
-      T extends number[] ? AnyArray<NumberSchemaDefinition> | AnyArray<SchemaTypeOptions<number>> :
-      T extends boolean[] ? AnyArray<BooleanSchemaDefinition> | AnyArray<SchemaTypeOptions<boolean>> :
-      T extends Function[] ? AnyArray<Function | string> | AnyArray<SchemaTypeOptions<Unpacked<T>>> :
-      T | typeof SchemaType | Schema<any, any, any> | SchemaDefinition<T> | Function | AnyArray<Function>;
+        T extends boolean ? BooleanSchemaDefinition :
```
Comment (Uzlopak) [inline types/index.d.ts:946]: Kind of seems ugly and makes it less readable

Comment (Uzlopak) [reply types/index.d.ts:946]: Well... Probably neglectable.

Comment (AbdelrahmanHafez) [reply types/index.d.ts:946]: Agreed, in this instance, it seems like the incorrectly indented version looks better, but the benefit of having consistent indentation automatically outweighs this one instance.

**Review by vkarpov15 — APPROVED** [review]
Thanks :+1:

**Comment by AbdelrahmanHafez** [comment]
Happy to help 👍

**Pull request merged**